spec/customer_service_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe CustomerService do
+  let(:twilio)   { class_double(Twilio, :twilio) }
+  let(:client)   { double(:client, messages: messages) }
+  let(:messages) { double(:messages, create: 'text has been sent') }
+  subject        { described_class.new(twilio) }
+
+  describe 'confirmation_text' do
+    it 'sends customers texts' do
+      allow(subject).to receive(:create_client) { client }
+      expect(subject.confirmation_text).to eq 'text has been sent'
+    end
+
+    it 'calls create client method on self' do
```

Comment:
impressed that you got round to actually testing twilio itself, I did not reach that point!